From dd8e684e01f18e812fb7d22a33ea186b4622b7a6 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Tue, 15 Jan 2019 11:51:18 -0800
Subject: [PATCH 2276/2278] net: sched: rename qdisc_put() to qdisc_destroy()

just change qdisc_put() to qdisc_destroy() in a sch_taprio.c file.
This is more easier approach instead of using the following patch.

net: sched: rename qdisc_destroy() to qdisc_put()
Current implementation of qdisc_destroy() decrements Qdisc reference
counter and only actually destroy Qdisc if reference counter value reached
zero. Rename qdisc_destroy() to qdisc_put() in order for it to better
describe the way in which this function currently implemented and used.

Extract code that deallocates Qdisc into new private qdisc_destroy()
function. It is intended to be shared between regular qdisc_put() and its
unlocked version that is introduced in next patch in this series.

Signed-off-by: Vlad Buslov <vladbu@mellanox.com>
Acked-by: Jiri Pirko <jiri@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 net/sched/sch_taprio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/sched/sch_taprio.c b/net/sched/sch_taprio.c
index 206e4db..1754c75 100644
--- a/net/sched/sch_taprio.c
+++ b/net/sched/sch_taprio.c
@@ -685,7 +685,7 @@ static void taprio_destroy(struct Qdisc *sch)
 
 	if (q->qdiscs) {
 		for (i = 0; i < dev->num_tx_queues && q->qdiscs[i]; i++)
-			qdisc_put(q->qdiscs[i]);
+			qdisc_destroy(q->qdiscs[i]);
 
 		kfree(q->qdiscs);
 	}
-- 
2.7.4

